packages/ui/src/pages/Home.tsx

```diff
@@ -105,12 +106,7 @@ const Home = ({ className }: Props) => {
         <h1>Multix is an interface to easily manage complex multisigs.</h1>
         <p>Connect an extension to interact with Multix or watch an address.</p>
         <Button onClick={allowConnectionToExtension}>Connect Wallet</Button> or
-        <Button
-          component={Link}
-          to="/settings"
-        >
-          Watch an address
-        </Button>
+        <RouterLink to="/settings">Watch an address</RouterLink>
```

Comment:
Based on our conversation, you can disregard this comment, we can keep the link and button. Just please check the margin :)

Reply:
@Lykhoyda can you add a `margin-left: 1rem` to the left button so that it doesn't look as weird? It'd look like this
![image](https://github.com/ChainSafe/Multix/assets/33178835/06db076c-8b0d-49a7-9a5d-7603ea2ed29c)